api/swagger-spec/v1.json

```diff
@@ -14966,6 +14966,9 @@
    "v1.ConfigMapList": {
     "id": "v1.ConfigMapList",
     "description": "ConfigMapList is a resource containing a list of ConfigMap objects.",
+    "required": [
+     "items"
```

Comment:
I don't think so.  

The current behavior was considered broken by our existing client code that had expected the Kubernetes API to follow a consistent behavior and not have a nil items member in a JSON response.

Since clients need to support the non-nil response, and support iteration in that list, I don't see how a client can break from this change, but I know clients can break from our current behavior.